### DISCUSSION TITLE
Gaming and iphone

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#972] [General]
Hi guys! Since i started playing games on my iphone 13 i noticed it became kinda slow in performance.
Is it normal? Is there something i can do about it?

**Comment by MirkosMangle** [comment]
That is not normal but happens all the time actually

**Comment by ellielo** [comment]
Do you have enough memory on your device? Try clearing out some apps and data. The would be my first step probably. For other ideas and solutions , check out this blog - [https://applenapps.com/make-iphone-faster-for-gaming/](url).  The other important step - always update your phone when necessary.
